### PR TITLE
Allow making account deletion requests(no useEffect)

### PR DESCRIPTION
**Pull request by keikari**
Wanted to try the "right" way.  https://github.com/OdyseeTeam/odysee-frontend/pull/2929#discussion_r1307007099

Some changes to the original example. Main one being to allow running the `doAccountDeletionSequence` further even if account is already pending deletion.
Reasoning for this are synced, but unclaimed YouTube channels. Those wouldn't get wiped with `txo_spend`.  So users will have a way to remove them after claiming later.  
(Script used to actually delete the account will check if the account has any channels(including synced ones).  And will email notify user about that, also warning about not being able to resync once deleted.)  

(Also changed the `leftOverForFee`/wallet-empty-limit to 0.005, as that will make wallet at top show "Your Wallet"-text instead of 0.01. And probably still enough for almost all accounts to pass the sequence on first try.)   

* [x]  Change address

### CODE REVIEW

File: ui/modal/modalRemoveAccount/thunk.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          return 'error_occurred';
+        }
+      }
+    }
+
+    try {
+      const totalBalance = selectTotalBalance(state);
+      const isWalletEmpty = totalBalance <= 0.005; // Allows avoiding some possible issues with txo_spend and sending credits
+      if (!isWalletEmpty) {
+        await dispatch(doSpendEverything());
```
[the remark this answers is not in the document]
Comment (keikari) [reply ui/modal/modalRemoveAccount/thunk.js:42]: Went with the 2 
Where these reports can be seen?

(My original plan for unexpected issues was that if pressing the button keeps erroring on some account, it hopefully will also happen if I try to click it for them. But yeah, better to have that too)

[the remark this answers is not in the document]
Comment (keikari) [reply ui/modal/modalRemoveAccount/thunk.js:42]: Strange, when trying here the messages don't appear there.  (Made sure that the line is run by adding `console.log`s before/after it).

[the remark this answers is not in the document]
Comment (keikari) [reply ui/modal/modalRemoveAccount/thunk.js:42]: Ah, that worked, thanks!

**Comment by keikari** [comment]
Oops, forgot users without stripe accounts when testing removing of the card. Need to try again.

**Comment by keikari** [comment]
Would it be possible to get this merged in the current state, even though the deletion of some stripe stuff is missing?
This is still improvement over current account deletion process.

**Comment by tzarebczan** [comment]
Will take another walk through this next week and we can merge it. Didn't get to do the delete apis (both customer and account, but account more important probably).

**Pull request merged**